dora/core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -401,9 +408,13 @@ default URIStatus getStatus(AlluxioURI path)
    * @return the {@link URIStatus} of the file
    * @throws FileDoesNotExistException if the path does not exist
    */
+  @Deprecated
   URIStatus getStatus(AlluxioURI path, GetStatusPOptions options)
```

Comment:
For all the existing methods like `getStatus(AlluxioURI)`, let's keep those methods. There is probably no point in marking them `@Deprecated`. We can omit those `@Deprecated` for now.